boards/hifive1/src/main.rs

```diff
@@ -96,6 +96,14 @@ pub unsafe fn reset_handler() {
 
     let board_kernel = static_init!(kernel::Kernel, kernel::Kernel::new(&PROCESSES));
 
+    let dynamic_deferred_call_clients =
+        static_init!([DynamicDeferredCallClientState; 1], Default::default());
```

Comment:
It should get 2. I put it to 1 because it didn't have enough memory and wanted to check that this wasn't the cause (I eventually changed the stack allocation), and didn't put it back.

Reply:
Fixed.